package/network/services/hostapd/patches/032-mesh-implement-use-of-VHT20-config-in-mesh-mode.patch

```diff
@@ -68,15 +68,17 @@ Signed-off-by: Peter Oh <peter.oh@bowerswilkins.com>
  	wpa_printf(MSG_DEBUG,
  		   "IBSS/mesh: setup freq channel %d, sec_channel_offset %d",
  		   freq->channel, freq->sec_channel_offset);
-@@ -2314,7 +2319,10 @@ void ibss_mesh_setup_freq(struct wpa_sup
+@@ -2314,7 +2319,12 @@ void ibss_mesh_setup_freq(struct wpa_sup
  		}
  	} else if (ssid->max_oper_chwidth == VHT_CHANWIDTH_USE_HT) {
  		chwidth = VHT_CHANWIDTH_USE_HT;
 -		seg0 = vht80[j] + 2;
++#ifdef CONFIG_HT_OVERRIDES
 +		if (ssid->disable_ht40)
 +			seg0 = 0;
 +		else
 +			seg0 = vht80[j] + 2;
```

Comment:
I am not very familiar with the code, but is it ok to net set seg0 here at all when CONFIG_HT_OVERRIDES is not set?
I would suggest something like this:
```
#ifdef CONFIG_HT_OVERRIDES
		if (ssid->disable_ht40)
			seg0 = 0;
		else
#endif
			seg0 = vht80[j] + 2;
```